setup.py

```diff
@@ -5,11 +5,17 @@
 
 ext_modules = [
     Extension('ffm', ['fastFM/ffm.pyx'],
-              libraries=['m', 'fastfm'],
+              libraries=['m'],#, 'fastfm'],
```

Comment:
`fastfm` should still be linked since `ffm2` implements only a subset of `fastfm` aka fastFM-core (we should improve the naming, it really starts to be confusing).